### ISSUE TITLE
Reduced circulating guanidinoacetic acid concentration

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11236]
**Preferred term label:**

**Synonyms**
Low serum GAA concentration

**Definition (free text, please give PubMed ID)**

PMID: 20301745 Biochemical findings : 
AGAT deficiency. **Low GAA levels** in urine, **plasma**, or CSF and low or low-normal creatine and creatinine levels in urine, plasma, or CSF

**Parent term (use hpo.jax.org/app)**
Abnormal circulating proteinogenic amino acid derivative concentration HP:0033108

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
L-Arginine:glycine amidinotransferase deficiency (AGAT deficiency) ORPHA:35704

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6001352
- label: Decreased circulating guanidinoacetic acid concentration
- parents: [Abnormal circulating proteinogenic amino acid derivative concentration (HP:0033108)](https://hpo.jax.org/app/browse/term/HP:0033108)
- pmid: PMID:20301745
- def: The concentration of guanidinoacetic acid in the blood circulation is below the lower limit of normal.
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11236
- synonyms: Reduced circulating guanidinoacetic acid concentration(NONE)|Low serum GAA concentration(NONE)
